modules/vaos/app/services/eps/base_service.rb

```diff
@@ -57,8 +57,45 @@ def check_for_eps_error!(response_data, response, method_name = nil)
       raise_eps_error(error_value, response)
     end
 
+    def handle_eps_error!(e, method_name)
+      error_context = {
+        service: 'EPS',
+        method: method_name,
+        error_class: e.class.name,
+        timestamp: Time.current.iso8601
+      }.merge(parse_eps_backend_fields(e.message.to_s)).compact
+
+      Rails.logger.error("#{CC_APPOINTMENTS}: EPS service error", error_context)
```

Comment:
this is the shared method that handles logging the error messages, lmk if any changes are needed such as the `service:` value or otherwise. I believe ours is the only part of vets-api that makes calls to EPS so maybe just leaving as is would be ok for debugging.